$OpenBSD: patch-configure,v 1.4 2015/09/22 08:11:24 semarie Exp $
Remove requirement for curl or wget.
The snapshot isn't downloaded but copied by post-configure.
--- configure.orig	Tue Aug  4 21:28:47 2015
+++ configure	Fri Aug  7 06:37:54 2015
@@ -696,7 +696,7 @@ putvar CFG_BOOTSTRAP_KEY
 
 step_msg "looking for build programs"
 
-probe_need CFG_CURLORWGET  curl wget
+probe CFG_CURLORWGET  curl wget
 if [ -z "$CFG_PYTHON_PROVIDED" ]; then
     probe_need CFG_PYTHON      python2.7 python2.6 python2 python
 fi
